etc/README.md

```diff
@@ -1,15 +1,31 @@
-## Builds
+# Builds
 
 Builds are run from a docker build image that is configured with the node and go we support.
 Our circle.yml uses this docker container to build, test and create release packages.
 
-### Updating new node/go versions
-After updating the Dockerfile_build run
+## Updating new node/go versions
 
-`docker build -t quay.io/influxdb/builder:chronograf-$(date "+%Y%m%d") -f Dockerfile_build .`
+Prereqs -- have python3.
```

Comment:
Herein used shell scripts do not execute python, so it is not IMHO required.  Should be simply

After updating the `Dockerfile_build` follow the steps below.

Reply:
removed... perhaps to build you need good luck, too